includes/notes/class-wc-payments-notes-additional-payment-methods.php

```diff
@@ -38,64 +38,8 @@ class WC_Payments_Notes_Additional_Payment_Methods {
 	 * Get the note.
 	 */
 	public static function get_note() {
-		// Show this notice only if UPE settings preview is disabled, and UPE flag is not enabled.
-		if ( false === WC_Payments_Features::is_upe_settings_preview_enabled() ) {
```

Comment:
Thoughts on also cleaning up the `maybe_enable_upe_feature_flag` and `set_account ` methods? For the `maybe_enable_upe_feature_flag` method, the body might just contain the redirect 🤷

Reply:
That's a good catch. I think `maybe_enable_upe_feature_flag` can be removed entirely because [its only invocation from `class-wc-payments.php`](https://github.com/Automattic/woocommerce-payments/blob/develop/includes/class-wc-payments.php#L1374C50-L1374C79). Same for `set_account`. Done in https://github.com/Automattic/woocommerce-payments/pull/7821/commits/cc3a3928b0f65a59e0b79beeacf86deb3062b46c (`maybe_enable_upe_feature_flag`) & https://github.com/Automattic/woocommerce-payments/pull/7821/commits/7c269f538dcd79509cdd045d0e1b7a97a0646457 (`set_account` + some unused constants)